### ISSUE TITLE
Age appropriate 'last updated' time measurements

### ISSUE THREAD

**Issue by maxelkins** [RaspberryPiFoundation/editor-ui#409]
Currently on the "Your project page" we have some not so age appropriate time measurements like 'qrt'
![image](https://user-images.githubusercontent.com/43481548/224305690-4de2e9df-6669-4d93-8cd1-e42a5fda20de.png)

One suggestion could be:
- Just now
- X seconds ago
- X minutes ago
- X hours ago
- X days ago
- X months ago
- X years ago
(And singular versions, i.e. '1 month ago')

Possibly an argument for shortening them, i.e 'min' vs 'minute' but unless its a space issue full words are probably clearer.

### Some options:
@patch0:
https://date-fns.org/v2.29.3/docs/formatDistanceToNow
instead of https://date-fns.org/v2.29.3/docs/intlFormatDistance

@maddyjoe:
- X seconds ago
- X minutes ago
- X hours ago
- Then date after that. i.e. DD/MM/YY

**Comment by sra405** [comment]
Solution should be applied everywhere
